init-haskell

```diff
@@ -1,22 +1,35 @@
 #! /usr/bin/env nix-shell
-#! nix-shell --pure --keep NIX_SSL_CERT_FILE --keep GITHUB_TOKEN -i bash -p git mustache-go niv direnv nix
+#! nix-shell --pure --keep NIX_SSL_CERT_FILE --keep GITHUB_TOKEN -i bash -p git mustache-go jq niv direnv nix
 set -Eeuxo pipefail
 
 # init-haskell location
 base=$(dirname "$(readlink -f "$0")")
 
+ghc=${1:-8.10.7}
```

Comment:
Thoughts on changing this to an env var `INIT_HASKELL_GHC` so people can set it in their profile (and we don't have to do arg parsing for further arguments)?